src/components/Sparkles/styles.module.scss

```diff
@@ -0,0 +1,35 @@
+%base {
+  position: absolute;
+  overflow: hidden;
+}
+
+%wrapper {
```

Comment:
it looks like maybe `wrapper` is now superfluous? if it's only used to extend base into star and add z-index, then you could do those two things directly in star

Reply:
Oh, there's a bug—`.tinyStar` should extend `%wrapper`